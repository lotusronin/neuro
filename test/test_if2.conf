conf = {
'src' : 'test_if2.nro',
'expected' : 'test_if2.expected'
}
