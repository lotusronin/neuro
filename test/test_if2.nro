import stdio;

fn main() : u32 {
    var1 := 5;
    var2 := 3;
    if(var1 > var2) {
        stdio::puts("var1 is greater!");
    }
    return 0;
}
