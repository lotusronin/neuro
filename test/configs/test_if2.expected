var1 is greater!
